pom.xml

```diff
@@ -201,10 +201,12 @@
     </dependencies>
 
     <properties>
-        <java-version>1.8</java-version>
-        <encoding>UTF-8</encoding>
+        <!-- == Maven Plugin Versions == -->
         <org.codehaus.mojo.build-helper-maven-plugin.version>1.9.1</org.codehaus.mojo.build-helper-maven-plugin.version>
         <maven-war-plugin.version>2.5</maven-war-plugin.version>
+        <!-- == Project Properties == -->
+        <java-version>1.8</java-version>
+        <encoding>UTF-8</encoding>
```

Comment:
- [ ] It is better to align the order of the property to `terasoluna-gfw-web-multi-blank `...